nativelib/src/main/resources/scala-native/gc/commix/Stats.c

```diff
@@ -70,28 +71,27 @@ void Stats_OnExit(Stats *stats) {
 #endif // ENABLE_GC_STATS
 
 #ifdef ENABLE_GC_STATS_SYNC
-void Stats_MarkStarted(Stats *stats) {
+void Stats_PhaseStarted(Stats *stats) {
     if (stats != NULL) {
-        stats->mark_waiting_start_ns = 0;
-        stats->mark_waiting_end_ns = 0;
+        stats->packet_waiting_start_ns = 0;
+        stats->packet_waiting_end_ns = 0;
     }
 }
-void Stats_MarkerGotFullPacket(Stats *stats, uint64_t end_ns) {
+void Stats_GotNotEmptyPacket(Stats *stats, uint64_t end_ns, eventType eType) {
```

Comment:
`eventType` is not used in this function

Reply:
Good catch, I forgot to test Commix with ENABLE_GC_STATS_SYNC (instead settling on ENABLE_GC_STATS), leading to this inconsistency. Now after adjustments I tested all GC_STATS modes.